nginx-core/tools/fix-permission.sh

```
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -eu
+
+echoColor() {
+  # 1: red, 2: green, 3: yellow, 4: blue, 5: purple, 6: cyan, 7: light gray
+  printf "\x1b[0;3$1m$2\x1b[0m"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)